vmdb/app/models/container_definition.rb

```diff
@@ -1,5 +1,5 @@
 class ContainerDefinition < ActiveRecord::Base
   # :name, :image, :image_pull_policy, :memory, :cpu
   belongs_to :container_group
-  has_many :container_port_configs
+  has_many :container_port_configs, :dependent => :destroy
```

Comment:
@Fryguy is this how it's supposed to be done or do we have some problem in the `save_inventory` part that is not capable of detecting what to remove?